docs/installation.rst

```diff
@@ -62,6 +62,47 @@ Install conda via anaconda or miniconda (no root privileges required)
         conda -c conda-forge boost
         conda -c anaconda cmake
 
+**Installing dependencies via spack**
+
+``spack`` is a package manager which was designed for HPC software. It is language agnostic and
+compiles binaries from source. See quote from ``spack``'s `documentation
+<https://spack.readthedocs.io/en/latest/>`_ below:
+
+   Spack is a package management tool designed to support multiple versions and configurations of
+   software on a wide variety of platforms and environments. It was designed for large supercomputing
+   centers, where many users and application teams share common installations of software on clusters
```

Comment:
Not sure whether this quote is encouraging or leads users to believe they don't need spack on their personal computer. Maybe leave out the "It was designed..." sentence would make it sound less scary?